Resources/Prototypes/Entities/Objects/Fun/spectral_locator.yml

```diff
@@ -0,0 +1,58 @@
+- type: entity
+  id: SpectralLocatorUnpowered
+  parent: BaseItem
+  name: spectral locator
+  description: Appears to be a modified anomaly locator. Seems very old.
+  suffix: Unpowered
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/spectrallocator.rsi
+    layers:
+      - state: icon
+      - state: screen
+        shader: unshaded
+        visible: false
+        map: ["enum.ToggleVisuals.Layer"]
+  - type: Appearance
+  - type: GenericVisualizer
+    visuals:
+      enum.ToggleVisuals.Toggled:
+        enum.ToggleVisuals.Layer:
+          True: { visible: true }
+          False: { visible: false }
+  - type: ItemToggle
+  - type: ProximityBeeper
+  - type: ProximityDetector
+    range: 12
+    criteria:
+      components:
+      - Spectral
```

Comment:
This is going to query every single entity in a 12 tile range, I don't think proximity detector code is suitable for this yet without causing performance issues.

Reply:
Doesn't the anomaly locator do the same on a much wider radius?

Reply:
yeah, and this is much rarer than anomaly locators so should probably be fine